pkg/master/master_test.go

```diff
@@ -378,7 +378,7 @@ func TestValidOpenAPISpec(t *testing.T) {
 	}
 
 	// TODO(mehdy): The actual validation part of these tests are timing out on jerkin but passing locally. Enable it after debugging timeout issue.
-	disableValidation := true
+	disableValidation := false
```

Comment:
You are right, but the test is not passing yet and until then, this PR is not ready for review.